tools/GenerateDocFxStructure/Program.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Web;
+using CommandLine;
+using GenerateDocFxStructure.Helpers;
+
+namespace GenerateDocFxStructure
+{
+    class Program
+    {
+        private static readonly string[] _mediaExtentions = new string[] { ".png", ".jpg", ".jpeg", ".mpg", ".mpeg", ".mov", ".mp4", ".bmp", ".gif", ".fzz" };
+        private static CommandlineOptions? _options;
+        private static int _returnvalue;
+        private static MessageHelper? _message;
+
+        private static List<string> allFiles = new List<string>();
+
+        private static int Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
```

Comment:
😄

Reply:
:-) :-) :-) I think I forgot to remove something :-D